openshift-ci/capz-test-env.sh

```
@@ -0,0 +1,85 @@
+export DEPLOYMENT_TIMEOUT
+
+# Randomize NAME_PREFIX per run to avoid Azure Key Vault name collisions.
+# KV names are globally unique with mandatory soft-delete — reusing a static
+# name fails with VaultAlreadyExists if a previous run's vault wasn't purged.
+# The first step to source this file generates the suffix; subsequent steps
+# read the same value from SHARED_DIR.
+# NOTE: NAME_PREFIX must be ≤11 chars because ARO HCP node pool names
+# (${NAME_PREFIX}-mp1) are limited to 15 characters.
+NAME_PREFIX_FILE="${SHARED_DIR:-/tmp}/name-prefix"
+if [[ -f "$NAME_PREFIX_FILE" ]]; then
+  export NAME_PREFIX=$(cat "$NAME_PREFIX_FILE")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)